server/src/main/java/com/google/musicanalysis/types/AnalysisGroup.java

```diff
@@ -2,8 +2,9 @@
 
 import java.util.ArrayList;
```

Comment:
add quick one line comment that explains how `AnalysisGroup`, `NLPResult`, `NameAndChannel` are going to be used in each .java file.